scripts/destroy-data-mesh.sh

```diff
@@ -51,7 +52,14 @@ ccloud::validate_ccloud_config $CONFIG_FILE || exit 1
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 ccloud::generate_configs $CONFIG_FILE > /dev/null
 source delta_configs/env.delta
-SERVICE_ACCOUNT_ID=$(ccloud::get_service_account $CLOUD_KEY) || exit 1
+SERVICE_ACCOUNT_ID=$(ccloud::get_service_account $CLOUD_KEY)
+
+if [ -z "${SERVICE_ACCOUNT_ID+x}" ]; then
```

Comment:
is the `+x` intentionally there?

Reply:
Yep, that was intentional. In this case it's mostly moot since a service_account_id will always be a non-zero length string, or else unset. 

${var+x} is a [parameter expansion](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02) which evaluates to nothing if var is unset, and substitutes the string x otherwise.